### PR TITLE
rename some files/dirs

### PR DESCRIPTION
**Pull request by amnona**

### CODE REVIEW

File: calour/__init__.py [hunk 1]
```diff
@@ -8,3 +8,8 @@
 
 __credits__ = "https://github.com/biocore/calour/graphs/contributors"
 __version__ = "0.1.0.dev0"
+
+from calour.experiment import *
+from calour.io import *
+from calour.filtering import *
+from calour.sorting import *
```
Comment (RNAer) [inline calour/__init__.py]: I don't think we need these imports

File: calour/experiment.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
 import scipy
 import biom
 
+import calour as ca
+
```
Comment (RNAer) [inline calour/experiment.py]: could you remove the changes of `experiment.py` for now? It will conflict with my changes.

File: calour/filtering.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # calour functions for filtering samples/observations
 # functions should call reorder_samples() / reorder_obs()
 
+import calour as ca
+
```
Comment (RNAer) [inline calour/filtering.py]: use `from .experiment import Experiment` instead.

File: calour/__init__.py (hunk 1 above)
Comment (amnona) [inline calour/__init__.py]: i think this is like in numpy
we import all functions from all modules so they are all available under calour.XXX
this is so the user does not need to remember in which module is function is located
so when you use calour, you can just do import calour and then use whatever function you want...

File: calour/experiment.py (hunk 2 above)
Comment (amnona) [inline calour/experiment.py]: this made calour work (so i can use the read and test the functions i'm writing...)
maybe you/i can merge the PRs and resolve the conflicts?
what can we do?

File: calour/filtering.py (hunk 3 above)
Comment (amnona) [inline calour/filtering.py]: i want to have all the calour functions under one place (like importing numpy...) why not do it?

File: calour/__init__.py (hunk 1 above)
Comment (RNAer) [inline calour/__init__.py]: with dynamically import functions into class methods, we only need to import the class to access them.

File: calour/experiment.py (hunk 2 above)
Comment (RNAer) [inline calour/experiment.py]: it is probably easier to just revert the changes for the time being. saw you just merged my PR. maybe update based on that?

File: calour/__init__.py (hunk 1 above)
Comment (amnona) [inline calour/__init__.py]: right :)
is there an elegant way to import all functions into the class (i.e. without explicitly writing each function name?)

**Comment by RNAer** [comment]
I already did it in the PR you merged :-)

File: calour/__init__.py (hunk 1 above)
Comment (wasade) [inline calour/__init__.py]: please do not do `import *`, it greatly complicates understanding codebases.

File: calour/experiment.py [hunk 4]
```diff
@@ -44,19 +46,33 @@ def __init__(self, data, sample_metadata, feature_metadata=None,
...
 
+        # flag if data array is sparse (True) or dense (False)
+        self.sparse = sparse
+
     def __repr__(self):
         '''
         print the information about the experiment
         should have number of samples, observations, first 3 sequences and first 3 samples?
         '''
+        return('Experiment %s with %d samples, %d features' % (self.description,self.data.shape[0],self.data.shape[1]))
```
Comment (wasade) [inline calour/experiment.py]: please make sure flake8 is being executed by travis

File: calour/normalization.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    -------
+    newexp : Experiment
+        the normalized experiment
+    '''
+    if not inplace:
+        exp = deepcopy(exp)
+
+    normfactor = reads / np.sum(exp.data, axis=axis)
+    if axis==0:
+        exp.data = exp.data * normfactor[:, None]
```
Comment (wasade) [inline calour/normalization.py]: both branches are the same?

**Comment by wasade** [comment]
Are there tests?
